### DISCUSSION TITLE
Charts are awesome!

### DISCUSSION THREAD

**Discussion by akaszynski** [pyvista/pyvista#2457] [Show and tell]
Just wanted to thank @dcbr for #1432 and all his hard work. I'm using `pyvista` in a closed source application and have found charts to be a great way to visualize graphs in real time within the VTK render window. It's awesome, and even updates in real-time.

![image](https://user-images.githubusercontent.com/11981631/162982282-1f124b66-6c0c-4138-b57f-d24f31350872.png)

**Comment by dcbr** [comment]
Super glad you like it :) These were actually the kind of applications I had in mind while working on it (i.e. 3D visuals with 2D chart overlays for more information), so it's great to see it being used in downstream projects!
